Assignment1.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+import plotly.express as px
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import classification_report, confusion_matrix
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.svm import SVC
+from sklearn.pipeline import Pipeline
+
+#Loading data into Pandas
+df = pd.read_csv('https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data',
+                 names=["SepalLengthCm", "SepalWidthCm", "PetalLengthCm", "PetalWidthCm", "Species"])
+print(df)
+
+#Summarizing stats
+print(df.describe())
+
+
+#Some Random Figures 
+fig = px.scatter(df, x="SepalWidthCm", y="SepalLengthCm", color="Species", size='PetalLengthCm',
+                 hover_data=['PetalWidthCm'])
+
+fig2 = px.bar(df, x="SepalWidthCm", y="SepalLengthCm", color="Species")
+
+fig3 = px.box(df, y="PetalWidthCm", color="Species", points='all')
+
+fig4 = px.violin(df, y="PetalLengthCm", color="Species", violinmode='overlay', hover_data=df.columns)
+
+fig5 = px.ecdf(df, x="SepalLengthCm", y="SepalWidthCm", color="Species", ecdfnorm=None)
+
```

Comment:
do fig1.show(),fig2.show(), etc so we can see the plots. the code runs fine but they don't show up unless you have them

Reply:
Thanks for the note! I commented fig.show() to the code.